src/domain/courseSearch/Search.tsx

```diff
@@ -120,9 +129,14 @@ const Search: React.FC<Props> = ({ scrollToResultList }) => {
     setSelectedPlaces(places);
     setSelectedTexts(text);
     setSelectedDateTypes(dateTypes);
-    setMinAgeInput(audienceMinAgeGt || '');
-    setMaxAgeInput(audienceMaxAgeLt || '');
-
+    // setMinAgeInput(audienceMinAgeGt || '');
+    // setMaxAgeInput(audienceMaxAgeLt || '');
+    if (suitableFor?.length) {
```

Comment:
Jep! I forgot that it's fail safe to check the values with out of bound index in JS. -_-

Reply:
actually I had chosen this because suitableFor filter could have been undefined also. I changed it so that it will always return a list like the other list filters does too.

Reply:
`if (suitableFor?.[0])`